src/sourmash/tax/tax_utils.py

```diff
@@ -1482,3 +1482,519 @@ def load(cls, locations, **kwargs):
                 tax_assign.add(this_tax_assign)
 
         return tax_assign
+
+
+# strategy from: https://subscription.packtpub.com/book/programming/9781800207455/10/ch10lvl1sec01/using-dataclasses-to-simplify-working-with-csv-files
+@dataclass
+class GatherRow(): # all cols should match "gather_write_cols" in `search.py`
```

Comment:
```suggestion
class GatherRow: # all cols should match "gather_write_cols" in `search.py`
```
:)

Reply:
done in https://github.com/sourmash-bio/sourmash/pull/2446, so can I leave this here to avoid conflicts?

Reply:
yep sounds good